package.json

```diff
@@ -124,13 +124,16 @@
     "InstantMessaging"
   ],
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^1.10.2",
+    "@typescript-eslint/parser": "^1.10.2",
```

Comment:
We'd want exact version numbers here, right?
Forgot to comment this the last time around.

Reply:
Done during rebase!